### DISCUSSION TITLE
시간 입력 어떻게 받음??

### DISCUSSION THREAD

**Discussion by MadeByZiNo** [Goranies/ProjectRepository#35] [Q&A]
13시 30분 을 받는다고치면

13.5 이런식으로 받을지

어떻게 할지

**Comment by MadeByZiNo** [comment]
아요일도

**Comment by byungmeo** [comment]
마음대로
startHour = 13
startMinute = 30

endHour = 15
endMinute = 50

이렇게 해도 되고

나중에 실제로 응용할때 편한걸로 바꾸면 될듯
